Dev

IlyaKorastylev opened the pull request.

IlyaKorastylev committed:
В проект добавлена разметка главной страницы и страницы настроек.

IlyaKorastylev committed:
Реализован вывод тостов по нажатию на кнопки на экране MainActivity

IlyaKorastylev committed:
Реализована навигация по Activity, возврат из SettingsActivity.kt на MainActivity.kt.
Также в activity_settings.xml MaterialButton тёмной темы заменён на Switch.

gusar-off commented on app/src/main/java/com/example/playlistmaker/SettingsActivity.kt:
```diff
@@ -0,0 +1,20 @@
+
+        searchBackButton.setOnClickListener {
+            val searchBackIntent = Intent(this, MainActivity::class.java)
+            startActivity(searchBackIntent)
```
Для возврата назад лучше завершать текущее активити вызвав метод `finish()`, иначе будет засоряться стек навигации

gusar-off commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -1,18 +1,81 @@
+        android:letterSpacing="0"
+        android:textSize="@dimen/base_text_size"
+        android:textColor="@color/text_and_icon_color"
+        android:layout_marginTop="16dp"
```
Тут не все повторяющиеся отступы/размеры вынесены в ресурсы dimens

gusar-off commented on app/src/main/res/layout/activity_settings.xml:
```diff
@@ -0,0 +1,100 @@
+            android:layout_height="wrap_content"
+            android:src="@drawable/arrow_back"
+            android:layout_marginTop="20dp"
+            android:layout_marginStart="20dp" />
```
И тут

IlyaKorastylev committed:
В dimens.xml вынесены повторяющиеся размеры.
В SettingsActivity.kt в методе setOnClickListener добавлен finish() для закрытия текущей activity.
